### PR TITLE
176162207 feedback settings

### PR DESCRIPTION
**Pull request by mklewandowski**
This PR sets the activity and question feedback settings when we display the activity and question feedback panels.  We must set the feedback type (text in both cases) before we can start providing feedback to the students.

### CODE REVIEW

File: js/containers/portal-dashboard/feedback/question-feedback-panel.tsx [hunk 1]
```diff
@@ -62,6 +72,17 @@ class QuestionFeedbackPanel extends React.PureComponent<IProps> {
...
   }
+
+  private updateQuestionFeedbackSettings = () => {
+    const questions = this.props.activity.get("questions");
+    questions.forEach((question: any) => {
+      const questionId = question.get("id");
+      if (!this.props.settings.get("questionSettings")?.get(questionId)?.get("feedbackEnabled")) {
+        this.props.updateQuestionFeedbackSettings(questionId, { feedbackEnabled: true });
+      }
+      this.props.updateQuestionFeedbackSettings(questionId, { feedbackEnabled: true });
```
Comment (emcelroy) [inline js/containers/portal-dashboard/feedback/question-feedback-panel.tsx:83]: Is line 83 needed? Looks like it negates the need for lines 80-82.

Comment (mklewandowski) [reply js/containers/portal-dashboard/feedback/question-feedback-panel.tsx:83]: Yup.  My mistake.  Thanks for catching this!

Comment (mklewandowski) [reply js/containers/portal-dashboard/feedback/question-feedback-panel.tsx]: Fixed.

**Review by emcelroy — APPROVED** [review]
Looks good!

**Pull request merged**